gulp/DevelopmentTasks.ts

```diff
@@ -145,7 +145,14 @@ const doc: ITaskFunction = async () => {
             commandNameSummary += " (experimental)";
         }
 
-        const anchorTag = "module-" + definition.name;
+        // create anchor tag that matches full command
+        let anchorTag = "";
+        if (oldCommandName !== "") {
+            anchorTag = `${oldCommandName}-${definition.name}`;
```

Comment:
I believe we need to trim the `oldCommandName` since a space is added below

Reply:
Ah I missed this. Will trim the anchor tags as they are passed an an argument because the examples rely on the space to when inserting `zowe` before the command.

Reply:
Resolved by 1d4ef12